### DISCUSSION TITLE
Post your Flexi Rex makes here!

### DISCUSSION THREAD

**Discussion by DrLex0** [DrLex0/print3D-FlexiRex#3] [Show and tell]
## 👋 Welcome!
This is intended to be an alternative to Thingiverse's “makes.” You can upload photos of your print(s) of this model here. Some guidelines:
* Some info about what printer you used, what filament, and print settings, would be nice.
* This is not a megapixel contest. A photo of 1280×960 resolution or up to half a MegaByte is sufficient, and doesn't choke everyone's networks.

**Comment by grawlinson** [comment]
![IMG_9972](https://github.com/DrLex0/print3D-FlexiRex/assets/4408051/fd3a92cd-e54f-4e9b-a005-6deede536522)

Bottom one printed at 270% scale on a Bambu X1C using PETG with additional parameters in #6. Haven't quite tuned the printer yet, as shown by the imperfect joints.

**Comment by tooshel** [comment]
I have printed 1000s of them. Not even exaggerating. 984 at one time for a parade. I have printed them at all different sizes from the tiny, but still flexi 50% all the way up to 360%. Here is just a sample of pictures I have taken of them over the years. Obviously, I love this model and I'm glad you shared it with the world! Also note that I haven't charged for 99.9% of them . . . and that 0.1% was just people paying me back for the filament on huge ones. 

![PXL_20221119_061229631](https://github.com/DrLex0/print3D-FlexiRex/assets/223338/7973cab4-a2be-40ce-97a5-077d8995912f)
![PXL_20230109_061116584](https://github.com/DrLex0/print3D-FlexiRex/assets/223338/ed0737a8-559d-44d2-82f1-3c9a1117346e)
![PXL_20230306_230423875](https://github.com/DrLex0/print3D-FlexiRex/assets/223338/d3fb3f83-5b41-4b8b-a075-f3b1635021e4)
![PXL_20230705_060143569](https://github.com/DrLex0/print3D-FlexiRex/assets/223338/d2194bc3-4d57-4057-bff9-89558cf4af3a)
![PXL_20230531_050712768](https://github.com/DrLex0/print3D-FlexiRex/assets/223338/1d4b7918-9a17-4c30-a9f0-2a429e3284de)
![PXL_20230314_173549623](https://github.com/DrLex0/print3D-FlexiRex/assets/223338/c05212b8-19ea-4f77-896a-042fc216d9b1)
![PXL_20230531_050520548](https://github.com/DrLex0/print3D-FlexiRex/assets/223338/bec92f6a-fd56-4195-85e2-4441aa8cd6a8)

**Reply by GoatTech-42** [reply]
Wow, that's amazing!

**Comment by GoatTech-42** [comment]
This is the flexi rex off of printables before I saw the link to this repo, but I'll soon print the flexier rex. Thanks for this awesome model!
![thumbnail_image0](https://github.com/DrLex0/print3D-FlexiRex/assets/156948079/293b2b8c-18a2-42b3-b3c7-3448844797cb)
![thumbnail_image1](https://github.com/DrLex0/print3D-FlexiRex/assets/156948079/4f3d5adb-b230-4d3f-9b46-03e93b55f6a7)
![thumbnail_image2](https://github.com/DrLex0/print3D-FlexiRex/assets/156948079/5358819b-d4ee-430b-a82c-3dbcf693d644)
![thumbnail_image3](https://github.com/DrLex0/print3D-FlexiRex/assets/156948079/2b0f0f5c-0d88-400d-84b6-a734abc34128)
![thumbnail_image4](https://github.com/DrLex0/print3D-FlexiRex/assets/156948079/b3df8e9b-9c18-45c1-8299-81f02113186c)
_________Specs and Stuff_________
Printer: 2 Week-Old Ender 3 V3 SE
Slicer: Default Creality Print
Filament: Creality Hyper PLA Black
Settings: Default for the Slicer
Print time: about an hour at 100 MS
Temperature: 190C

**Comment by tooshel** [comment]
Got a Prusa XL recently and of course all I want to print is Flexerex. We left a few along the golden gate bridge, made a special one for post malone/taylor swift new song fortnite, left them at a google store and in several places all around that area. Also gave away almost 100 to fellow GDG members for Google I/O. 

![flexirex post taylor PXL_20240509_174033738 MP](https://github.com/DrLex0/print3D-FlexiRex/assets/223338/60d68276-0d9f-41ae-8225-2ebbbcc7e787)

![flexirex hanging out at the google store in mountain view PXL_20240513_205731644](https://github.com/DrLex0/print3D-FlexiRex/assets/223338/72684bf8-dc13-4f1d-bb4c-211e5a8c0b72)

![flexirex at the golden gate bridge IMG_6266](https://github.com/DrLex0/print3D-FlexiRex/assets/223338/c4b4035c-9f3a-4af3-89d9-17e4c557df05)

**Reply by GoatTech-42** [reply]
That's so cool!

**Reply by tooshel** [reply]
Thanks! It's by far my favorite thing to make and it's so fun seeing all the other cool thinks people do. Like the Santa Hat! I have printed 100s of those too! I need to find pictures of that one. And I wanted to make a graduation cap but just haven't gotten around to it.

**Reply by GoatTech-42** [reply]
Can you share the link so I can check it out too?

**Reply by tooshel** [reply]
You mean for the Post Malone/Taylor Swift one? I didn't post that anywhere . . . I want to make it better! That was just an experiment my daughter and I did in an hour or so of messing around converting images of the tatoos into STLs that I was resizing and trying to align in Tinkercad.

**Reply by GoatTech-42** [reply]
The santa hat :) Although the taylor swift one is pretty cool too!

**Reply by tooshel** [reply]
https://www.thingiverse.com/thing:3252216 . . . what's crazy is first time I saw it was on etsy and I wondered if it was out there already. I printed a few hundred of those too. And printed red and green Flexirexs to decorate a tree! That I mounted to a bike trailer and drove it around downtown Mesa. I need to find that picture and post it too.

**Comment by tooshel** [comment]
Another great print this time inspired by a thingiverse remix and an unnamed companies video. (They deleted my comment off the video so I figure they would rather not have the attention!) FlexiRex is holding a box of chocolates! And the box is detachable! 

![image](https://github.com/user-attachments/assets/85ac3bcb-4eb5-4f24-95f8-bebf78219570)

https://bsky.app/profile/tooshel.bsky.social/post/3li4dxythw22l
